partEZ/database/migrations/2016_03_03_161039_fix_iid_event_items.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class FixIidEventItems extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        if (Schema::hasTable('event_list_items'))
+        {
+            Schema::drop('event_list_items');
+        }
+        
+        Schema::create('event_list_items', function (Blueprint $table) {
+            $table->increments('iid')->unsigned();
```

Comment:
can we not just modify the column as opposed to dropping the whole table? This will cause a loss of existing data. The idea of modifying the table is that we don't need to drop it.